net/travelmate/files/travelmate.sh

```diff
@@ -228,7 +228,13 @@ f_net()
 {
 	local IFS raw result
 
-	raw="$(${trm_fetch} --timeout=$((trm_maxwait/6)) "${trm_captiveurl}" -O /dev/null 2>&1 | tail -n 1)"
+	raw="$(${trm_fetch} --timeout=$((trm_maxwait/6)) "${trm_captiveurl}" -O /dev/null 2>&1)"
+	if [ "$(printf "%s" "${raw}" | grep -c "^Redirected")" = "1" ] 
+	then
+		raw="$(printf "%s" "${raw}" | grep "^Redirected")"
+	else
+		raw="$(printf "%s" "${raw}" | tail -n 1)"
+	fi
 	raw="$(printf "%s" "${raw//[\?\$\%\&\+\|\'\"\:\*\=\/]/ }")"
 	result="$(printf "%s" "${raw}" | awk '/^Failed to redirect|^Redirected/{printf "%s","net cp";exit}/^Download completed/{printf "%s","net ok";exit}/^Failed|Connection error/{printf "%s","net nok";exit}')"
 	if [ "${result}" = "net cp" ]
```

Comment:
Nope, does not work. Not sure what line 242 is supposed to do, but it does not work with 
```bash
raw="Redirected to /login?dst=http%3A%2F%2Fcaptive.apple.com%2F on 172.16.0.1"
```

This is why I replaced that line in my proposal with a simpler version that just add the domain name after the `on`.
```
result="$(printf "%s" "net cp : ${raw//*on /}")"
```